src/client/src/containers/DependencyInfo/index.tsx

```diff
@@ -119,9 +119,17 @@ class DependencyInfo extends React.Component<Props> {
       ? getSvg.getGreenCheckSvg()
       : getSvg.getWarningSvg();
 
+    const keyDownHandler = (event: React.KeyboardEvent<HTMLDivElement>) => {
+      if (event.keyCode === 13 || event.keyCode === 32) {
```

Comment:
Should we have 13 and 32 as constants somewhere so that it be clear for any other developer looking at this code as to what do they refer to?

Reply:
Yea, I think we can change ALL keyhandlers when we fix #817